code/game/machinery/buttons.dm

```diff
@@ -49,6 +49,12 @@
 
 	setup_device()
 
+/obj/machinery/button/Destroy()
+	// Let them dump on the ground.
+	device = null
+	board = null
```

Comment:
no side effects like dropping stuff, this should qdel null

Reply:
That's fine with me, I just was avoiding changing existing side-effects. Done.